doc/en/user/source/data/raster/gdal.rst

```diff
@@ -45,92 +45,98 @@ Moreover, in order for GeoServer to leverage these libraries, the GDAL (binary)
-There are a few things to be careful with as well as some extra steps if you are deploying on Windows.
+Taking a look at the ``wrapper.conf`` configuration file available inside the GeoServer installation (at ``bin/wrapper/wrapper.conf``), there is this useful entry::
 
-As stated above, we have multiple versions like MSVC2005, MSVC2008 and so on matching the Microsoft Visual C++ Redistributables. Depending on the version of the underlying operating system you'll have to pick up the right one. You can google around for the one you need. Also make sure you download the 32 bit version if you are using a 32 bit version of Windows or the 64 bit version (has a "-x64" suffix in the name of the zip file) if you are running a 64 bit version of Windows.
-Again, pick the one that matches your infrastructure.
-   
-Note on running GeoServer as a Service on Windows
-++++++++++++++++++++++++++++++++++++++++++++++++++
+    # Java Library Path (location of Wrapper.DLL or libwrapper.so)
+    wrapper.java.library.path.1=bin/wrapper/lib
 
-Note that if you downloaded an installed GeoServer as a Windows service you installed the 32 bit version.
+To allow the GDAL native DLLs to be loaded, you have two options:
 
-Simply deploying the GDAL ImageI/O-Ext native libraries in a location referred by the PATH environment variable (like, as an instance, the JDK/bin folder) doesn't allow GeoServer to leverage on GDAL, when run as a service. As a result, during the service startup, GeoServer log reports this worrisome message:
+#. Move the native DLLs to the referenced path (bin/wrapper/lib)
+#. Add a ``wrapper.java.library.path.2=path/where/you/deployed/nativelibs`` entry just after the ``wrapper.java.library.path1=bin/wrapper/lib`` line.
 
-*it.geosolutions.imageio.gdalframework.GDALUtilities loadGDAL
-WARNING: Native library load failed.java.lang.UnsatisfiedLinkError: no gdaljni in java.library.path*
+Linux packages and setup
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+For common LTS Linux distribution there are packages for GDAL and the associated Java bindings,
+e.g., on Ubuntu and derivatives you can install them using::
 
-Taking a look at the wrapper.conf configuration file available inside the GeoServer installation (at bin/wrapper/wrapper.conf), there is this useful entry:
+  sudo apt-get install gdal-bin libgdal-java
+  
+The libraries as installed above are already in the search path, so no extra setup is normally needed.
+In case setting up the ``GDAL_DATA`` is required to handle certain projections, it's normally found
+in ``/usr/share/gdal/<version>``, so you can execute the following prior to start GeoServer, e.g::
 
-# Java Library Path (location of Wrapper.DLL or libwrapper.so)
-wrapper.java.library.path.1=bin/wrapper/lib
+  export GDAL_DATA=/usr/share/gdal/<version>
+  
+In case you decide to build from sources instead, remember to run ``configure`` with ``--with-java``,
+and after the main build and install, get into the ``swig/java`` and run a build and install there.
```

Comment:
get into the swig/java and run a build and install there:
do we have/do we need some instructions for this step too?

Reply:
Hmm... I did not want to go into providing specific build instructions for GDAL, those should be in the GDAL documentation, not here. I'd link to https://trac.osgeo.org/gdal/wiki/GdalOgrInJavaBuildInstructions and https://trac.osgeo.org/gdal/wiki/GdalOgrInJavaBuildInstructionsUnix instead.

Reply:
yep. I was thinking about linking the GDAL doc indeed..